pkg/applier/applier.go

```diff
@@ -178,28 +175,3 @@ func (a *Applier) parseFiles(files []string) ([]*unstructured.Unstructured, erro
 
 	return resources, nil
 }
-
-type restClientGetter struct {
```

Comment:
I might be mistaken, but if I recall correctly, it was done on purpose to avoid some caching issue. But it could be outdated info